backend/.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+node_modules/
+db/users.db

```

Comment:
why this?

Reply:
This represents sample user's database
can use the production database here^^

Reply:
Oh cools
Approving!!!